src/dds/rs-dds-sensor-proxy.h

```diff
@@ -63,6 +63,7 @@ class dds_sensor_proxy : public software_sensor
     std::map< std::string, streaming_impl > _streaming_by_name;
 
     formats_converter _formats_converter;
+    stream_profiles _target_profiles;
```

Comment:
Please rename `_active_converted_profiles`.
"target profiles" is a term used inside the converted and is not applicable here. Also these are not all the profiles, just those set by open.

Reply:
Done